README.md

```diff
@@ -818,13 +818,20 @@ import { Router } from "wouter";
 
 const handleRequest = (req, res) => {
   // top-level Router is mandatory in SSR mode
+  // pass an optional context object to handle redirects on the server
+  const ssrContext = {};
   const prerendered = renderToString(
-    <Router ssrPath={req.path} ssrSearch={req.search}>
+    <Router ssrPath={req.path} ssrSearch={req.search} ssrContext={ssrContext}>
```

Comment:
How about a shorter name `ssrCtx`?

Reply:
Given the other props used in the above example (`ssrPath` and `ssrSearch`) IMO `Context` fits better then an abbreviation.  Not sure there's any benefit to being terse here.